### PR TITLE
Remove threshold line in side panel

### LINKED ISSUE #1664: The number of thresholds is confusing because it is combined and we have no way of telling people that
**Describe the bug**
The number of thresholds is confusing because it is combined and we have no way of telling people that.

**To Reproduce**
1. Go to 'Explore the map'
2. Click on 'Census tracts'
3. See error

**Expected behavior**
Remove the threshold line 
Remove the "Disadvantaged" from the categories line
(see screenshot) 

**Screenshots**
option_two

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by vim-usds**
# Purpose
- closes #1664 

# QA

### QA spec
- [x] threshold line is removed

# QA round 2

### QA spec
- [ ] remove disadvantaged (this seems so much cleaner, BTW)

### CODE REVIEW

**Comment by KameronKerger** [comment]
the threshold line is removed but can we also get the "Disadvantaged" removed from "Disadvantaged in x categories"??? thx @vim-usds

**Comment by vim-usds** [comment]
@KameronKerger 

Ok yes.

Should it say 

`in x categories` OR

`In x categories`

so capital or lower case I?

**Comment by KameronKerger** [comment]
lower case I think @vim-usds - thank you.

**Review by TomNUSDS — APPROVED** [review]
LGTM

File: client/src/data/copy/explore.tsx [hunk 1]
```diff
@@ -314,21 +314,21 @@ export const COMMUNITY = {
...
 
-export const numberOfThresholdsExceeded = (thresholds:number) => <FormattedMessage
-  id={'explore.map.page.side.panel.num.thresholds.exceeded'}
-  defaultMessage={`At or above {numberOfThresholdExceed, plural, one {# threshold} other {# thresholds}}`}
-  description={`Navigate to the explore the map page. When the map is in view, click on the map. The side panel will show how many thresholds are exceeded`}
-  values={{
-    numberOfThresholdExceed: thresholds,
-  }}
-/>;
+// export const numberOfThresholdsExceeded = (thresholds:number) => <FormattedMessage
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/data/copy/explore.tsx:324]: I'll add a comment here. We are temporarily commenting out as it might get placed back in.

**Pull request merged**